docs/samples/Microsoft.ML.Samples/Dynamic/ModelOperations/OnnxConversion.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Microsoft.ML;
+
+namespace Samples.Dynamic.ModelOperations
+{
+    public static class OnnxConversion
+    {
+        class PlattModelInput
+        {
+            public bool Label { get; set; }
+            public float Score { get; set; }
+        }
+
+        static IEnumerable<PlattModelInput> PlattGetData()
+        {
+            for (int i = 0; i < 100; i++)
+            {
+                yield return new PlattModelInput { Score = i, Label = i % 2 == 0 };
+            }
+        }
+        public static void Example()
+        {
+            
+            var mlContext = new MLContext(seed: 0);
+
+            //Get input data
+            IDataView data = mlContext.Data.LoadFromEnumerable(PlattGetData());
+
+            //Get estimator
+            var calibratorEstimator = mlContext.BinaryClassification.Calibrators.Platt();
+
+            //Get pipeline(calibratorTransformer) by calling Fit()
+            var calibratorTransformer = calibratorEstimator.Fit(data);
+
+            //What you need to convert ML.NET model to onnx model are pipeline and input data
+            //In default, the onnx conversion will generate the onnx file with the latest OpSet version
+            using (var stream = File.Create("sample_onnx_conversion_1.onnx"))
+                mlContext.Model.ConvertToOnnx(calibratorTransformer, data, stream);
```

Comment:
Would it be good to also add code to show how to run this onnx model with the OnnxTransformer?

I guess we already have a sample showing how to run OnnxTransformer, but for new users it might be useful to have it in here how to run the onnx model they've just created.